apps/api/src/app/channels/usecases/update-sms-settings/update-sms-settings.command.ts

```diff
@@ -15,5 +15,5 @@ class TwillioSettings {
 export class UpdateSmsSettingsCommand extends EnvironmentWithUserCommand {
   @IsDefined()
   @ValidateNested()
-  twillio: TwillioSettings;
+  twilio: TwilioSettings;
```

Comment:
Rollback this one for now like this:
```suggestion
  twillio: TwilioSettings;
```